tests/probes/yamlfilecontent/test_probes_yamlfilecontent_array.xml

```diff
@@ -76,11 +76,16 @@
   <states>
 
     <ind-def:yamlfilecontent_state version="1" id="oval:0:ste:1">
-      <ind-def:value_of datatype="string">secureforward-offcluster</ind-def:value_of>
+      <ind-def:result datatype="record">
```

Comment:
This is a UX regression, checks that a particular value is present in a particular place will be quite common, and they are somehow difficult, IIUC the `value_of` in `name` is basically a keyword.
Is it possible to somehow preserve the old approach? I would even consider a new object/test/state for the proposed extension.

Reply:
I think the `yamlfilecontent_item` could contain both `value_of` element and `result`. The `value_of` element would be used for scalars and `result` for maps. If that's going to happen the `result` should probably be renamed.

Reply:
This is possible in theory. Also the `result` elements could be renamed to `values`. Not sure if this polymorphic approach would be in the spirit of OVAL language, though.

Reply:
Good point. I skimmed through the specification and I can't find existing example of it. The win57_item, activedirectory57_item, ldap57_item or sql_57 return a single element.

Regarding the naming,  In sql57_item and wmi57_item it's called "result". In  activedirectory57_item and ldap57_item it's "value". I like more "value" because YAML path expressions are describing values in some files. But actually it doesn't matter.

Reply:
So, as no other probe does this polymorphic stuff, I think it would be better to stick with only record-type result. We already have enough problems with adoption of the probe.

But, `value` or `result`? Speak up!

Reply:
I side with `value` as well.

Reply:
Done.